Src/lib/drivers/button/button.c

```diff
@@ -7,12 +7,14 @@
 #include "button.h";
 #define MIN_PRESSED_TIME_BORDER 0
 #define MAX_PRESSED_TIME_BORDER 10000
```

Comment:
The same as below MS if it in miliseconds

Reply:
Done